mnefun/_stats.py

```diff
@@ -118,7 +118,7 @@ def _ht2_p(mu, sigma, n_ave, mu_other, sigma_other, n_ave_other, use_pinv):
     # sinv by the mean; a is the left 3 oris, b is right 3 oris, which get
     # collapsed; v is vertices and ... is our time points, which get
     # preserved):
-    T2 = np.einsum('va...,vab,bn...->v...', mu, sinv, mu)
+    T2 = np.einsum('va...,vab,vb...->v...', mu, sinv, mu)
     # Then convert T**2 for p (here, 3) variables and n DOF into F:
```

Comment:
I assume the original _signature_ signifies matrix multiplication? A*B*A Is this still the same notation?

Reply:
For einsum, any dimension not preserved in the output is multiplied and added to collapse it, so a basic matrix multiplication which could be written in einsum notation as `'ab,bc->ac'`. So this is a left and right matrix multiplication (or really a left vector-matrix and then a vector-vector multiplication reducing to a scalar or so)